django-backend/fecfiler/reports/views.py

```diff
@@ -53,9 +66,35 @@ class ReportViewSet(CommitteeOwnedViewMixin, ModelViewSet):
     in CommitteeOwnedViewMixin's implementation of get_queryset()
     """
 
-    queryset = Report.objects.annotate(
-        report_code_label=report_code_label_mapping
-    ).all()
+    whens = [When(form_type=k, then=Value(v)) for k, v in version_labels.items()]
+
+    queryset = (
+        Report.objects.annotate(report_code_label=report_code_label_mapping)
+        # alias fields used by the version_label annotation only. not part of payload
+        .alias(
```

Comment:
Changed this to from annotation() to alias() since the fields are only used by the annotation() below.